src/components/minting/SelectCoCreators.tsx

```diff
@@ -137,6 +139,30 @@ const FormContent: FunctionComponent<FormContentProps> = ({
         </Button>
       </Stack>
 
+      <Stack mt={ 1.5 } flexDirection="row">
+        <Typography variant="subtitle2">
+          For every additional collaborator who will receive royalties, the fee
+          to complete the minting process will increase by 1.5 ADA.
+        </Typography>
+        <Tooltip
+          title={
+            "This cost is increased with each additional collaborator because " +
+            "this ADA needs to travel with each portion of stream tokens in order " +
+            "to complete the split, which is an added extra cost for each transaction."
+          }
+        >
+          <IconButton sx={ { padding: 0 } }>
+            <HelpIcon
```

Comment:
Just a note, but can be addressed during on the mobile responsive effort, the position of the help icon looks out of place when below the md (900px) sizing.

Reply:
uhm, i didnt think it looked that bad, but added 8 px spacing 🫡

Reply:
Just looked cramped on my screen 😅 looks good!